Formula/o/octave.rb

```diff
@@ -35,7 +36,10 @@ class Octave < Formula
   end
 
   # Complete list of dependencies at https://wiki.octave.org/Building
+  depends_on "autoconf" => :build # for the patches
+  depends_on "automake" => :build # for the patches
   depends_on "gnu-sed" => :build # https://lists.gnu.org/archive/html/octave-maintainers/2016-09/msg00193.html
+  depends_on "libtool" => :build # for the patches
```

Comment:
Thanks for fixing this Qt 6.7 problem!

Out of curiosity: 

<img width="435" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/2618447/b0c2287a-b01c-449a-8e77-3f92158004e1">

This addition of `depends_on "libtool" => :build` seems redundant with the existing plain (non-`:build`) `depends_on "libtool"` further down. The main (not `on_linux`) depends_on list now looks like this to me:

```
  depends_on "libtool" => :build # for the patches
[...]
  depends_on "libtool"
```

Does the `:build` version of the depends_on do something special here? I thought that all the regular depends_on dependencies were also installed/available at build time, and the `:build` dependencies meant "only install this when doing a build of this formula, but don't install it when installing the package from a bottle.

Reply:
It's a mistake